docs/community_features.md

```diff
@@ -1093,6 +1093,15 @@ to each individual note onset. ([#1978])
 
 - As the UI and implementation is still experimental, a community setting has to be activated to access the `CHORD LIBRARY` keyboard. See the [Community Features Menu](#5-community-features-menu-aka-runtime-settings) for more information.
 
+#### 4.4.1.8 - Display Piano Keyboard Layout
+
+- ([#2983]) Provides a classic piano keyboard layout for Deluge.
```

Comment:
can you add the link to this PR at the bottom of the community features document so that this PR reference works?

Thx